README.md

```diff
@@ -8,7 +8,7 @@ GitHub CLI is available for repositories hosted on GitHub.com and GitHub Enterpr
 
 ## Documentation
 
-[See the manual][manual] for setup and usage instructions.
+[See the manual][manual] for usage instructions. Scroll down for [installation](#Installation).
```

Comment:
What do you think about:

"For installation options see below, for usage instructions see the manual."